Alan_DCMotorTest/Motor_code.ino

```diff
@@ -0,0 +1,43 @@
+#include <Wire.h>
+#include <Adafruit_MotorShield.h>
+#include "utility/Adafruit_MS_PWMServoDriver.h"
+
+// Create the motor shield object with the default I2C address
+Adafruit_MotorShield AFMS = Adafruit_MotorShield(); 
+
+// Connect two motors with the following terminals
+Adafruit_DCMotor *motor1 = AFMS.getMotor(1); // Motor connected to M1
+Adafruit_DCMotor *motor2 = AFMS.getMotor(2); // Motor connected to M2
+Adafruit_DCMotor *motor3 = AFMS.getMotor(3); // Motor connected to M3
+
+const int buttonPin = 2;  // the number of the pushbutton pin
```

Comment:
Could you add another button (2 buttons total) so that one button drives M1 and another drives M2? Basically I'm thinking of a skid-steer drive setup where each button actuates one side of the vehicle if that makes sense.

Reply:
can do